includes/blocks/podcast/markup.php

```diff
@@ -74,46 +84,90 @@
 					<?php the_title(); ?>
 				</h3>
 			<?php endif; ?>
-			<div class="wp-block-podcasting-podcast__show-details">
-				<?php if ( $attributes['displayShowTitle'] && ! empty( $show_name ) ) : ?>
-					<span class="wp-block-podcasting-podcast__title">
-						<?php echo esc_html( $show_name ); ?>
-					</span>
-				<?php endif; ?>
-				<?php if ( $attributes['displaySeasonNumber'] && ! empty( $season_number ) ) : ?>
-					<span class="wp-block-podcasting-podcast__season">
-						<?php esc_html_e( 'Season: ', 'simple-podcasting' ); ?>
-						<?php echo esc_html( $season_number ); ?>
-					</span>
-				<?php endif; ?>
-				<?php if ( $attributes['displayEpisodeNumber'] && ! empty( $episode_number ) ) : ?>
-					<span class="wp-block-podcasting-podcast__episode">
-						<?php esc_html_e( 'Episode: ', 'simple-podcasting' ); ?>
-						<?php echo esc_html( $episode_number ); ?>
-					</span>
-				<?php endif; ?>
-			</div>
-			<div class="wp-block-podcasting-podcast__show-details">
-				<?php if ( $attributes['displayDuration'] && ! empty( $duration ) ) : ?>
-					<span class="wp-block-podcasting-podcast__duration">
-						<?php esc_html_e( 'Listen Time: ', 'simple-podcasting' ); ?>
-						<?php echo esc_html( $duration ); ?>
-					</span>
-				<?php endif; ?>
-				<?php if ( $attributes['displayEpisodeType'] && ! empty( $episode_type ) && 'none' !== $episode_type ) : ?>
-					<span class="wp-block-podcasting-podcast__episode-type">
-						<?php esc_html_e( 'Episode type: ', 'simple-podcasting' ); ?>
-						<?php echo esc_html( $episode_type ); ?>
-					</span>
-				<?php endif; ?>
-				<?php if ( $attributes['displayExplicitBadge'] && ! empty( $explicit ) ) : ?>
-					<span class="wp-block-podcasting-podcast__explicit-badge">
-						<?php esc_html_e( 'Explicit: ', 'simple-podcasting' ); ?>
-						<?php echo esc_html( $explicit ); ?>
-					</span>
-				<?php endif; ?>
+			<div id="podcast-details" style="display: none;">
```

Comment:
The IDs will need to be unique (use `wp_unique_prefixed_id( 'podcast-details' )`) to avoid duplicate IDs on index pages that contain multiple blocks.
![Screenshot 2025-01-21 at 1 07 08 PM](https://github.com/user-attachments/assets/9843ce19-60b2-497d-ab09-f102e67a1c40)